CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 7.4.0
```

Comment:
since the bug was introduced in 7.3.0 and this is a bug fix PR I suggest releasing it as 7.3.2